app/views/course_user_data/sudo.html.erb

```diff
@@ -1,10 +1,51 @@
+<% content_for :javascripts do %>
+
+  <script type="application/javascript">
+    jQuery(function() {
+      /* match user name/email with cud_email */
+      /* escape_javascript prevents issues with backslashes in names, etc. */
+      userData = {
+          <% @usersEncoded.each do |k,v| %>
+            "<%= j k %>": "<%= v %>",
```

Comment:
I am doing `j k` instead of `j k.html_safe` as per production Autolab, since that defeat the point of `CGI.escapeHTML`. That is, it unescapes the escaped html.